components/resources/ai_chat_prompts.grdp

```diff
@@ -1,68 +1,71 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <grit-part>
-  <message name="IDS_AI_CHAT_ARTICLE_PROMPT_SEGMENT" translateable="false">Here is an article in &lt;article&gt; tags:
+
+<message name="IDS_AI_CHAT_CLAUDE_SYSTEM_MESSAGE" translateable="false">
```

Comment:
Aside from formatting, this still seems a bit different from the llama system message. It doens't mention the "socially unbiased and positive" part, or to specify factual non-false information. Are all the remaining differences intentional?

Reply:
It is intentional. The way that we prompt Llama is different from Claude. For Anthropic, there likely is an upstream system message that they have set on which they have trained their model with all the harmless and helpful policies (we don't have visibility into it). We are trusting them with that part, that's why I have not included those parts in the Claude system message. For Llama, there's not upstream system message and it's fully set/decided by us. So we need to be as specific as possible in all regards.

Other than the harmlessness, there should be a general alignment elsewhere: expected behaviours, tone, form, limits, etc. As for the formatting, Anthropic suggest the bulleted format in their examples, Llama uses inline text. This might be very minor, but worth sticking to it.